readme.html

```diff
@@ -106,7 +228,7 @@ <h3>Automation User</h3>
     <ul>
         <li>Set the <b>User Type</b> to <b>Automation</b></li>
         <li>Give the user a <b>Username</b> like &quot;Slack Automation&quot;</li>
-        <li>Set <b>Allowed IPs</b> to <b>127.0.0.1</b></li>
+        <li>For security reasons, accessing 127.0.0.1 is not allowed. Set <b>Allowed IPs</b> same as the <b>instance IP or any</b>. (eg : If instance IP is 10.1.18.123, set allowed IP also 10.1.18.123)</li>
```

Comment:
`any` should just be a short term Allowed IP for testing

Reply:
**any** is one of the value we can pass in the field of allowed IPs, Would enclose it in quotes.

Reply:
Done